vendor/github.com/google/go-github/v58/github/repos.go

```diff
@@ -1440,12 +1440,12 @@ func (s *RepositoriesService) GetBranch(ctx context.Context, owner, repo, branch
 	return b, newResponse(resp), err
 }
 
-// renameBranchRequest represents a request to rename a branch.
```

Comment:
Please revert the changes made to this file. This is not part of the Git Town source code, but an external dependency that was downloaded into this monorepo for faster and more reliable builds.

An easy way to revert this is running 

```
git checkout main vendor/github.com/google/go-github/v58/github/repos.go
```

Reply:
Yes you're right, missing that wrong changes, I've reverted this change